mabwiser/mab.py

```diff
@@ -812,12 +866,15 @@ def __init__(self,
             lp = _ThompsonSampling(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.binarizer)
         elif isinstance(learning_policy, LearningPolicy.UCB1):
             lp = _UCB1(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.alpha)
+        elif isinstance(learning_policy, LearningPolicy.LinGreedy):
+            lp = _Linear(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.l2_lambda,
+                         0, learning_policy.epsilon, "ridge", learning_policy.arm_to_scaler)
         elif isinstance(learning_policy, LearningPolicy.LinTS):
             lp = _Linear(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.l2_lambda,
-                         learning_policy.alpha, "ts", learning_policy.arm_to_scaler)
+                         learning_policy.alpha, 0, "ts", learning_policy.arm_to_scaler)
```

Comment:
Similarly, you are setting `epsilon` to zero for LinTS and LinUCB?

Reply:
Yes, so that in predict_context() it will skip the random arm operations.